app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (OutdatedVaccineError,
```

Comment:
if I remember well between imports of different types of libs we have to insert 2 empty lines